test/autoCompletion.test.ts

```diff
@@ -2364,10 +2364,44 @@ describe('Auto Completion Tests', () => {
       });
 
       const content = 'test:\n  - and\n  - - ';
-      const completion = await parseSetup(content, 19);
+
+      const completion = await parseSetup(content, 20);
```

Comment:
The previous index 19 is probably not correct. index 20 is the end of the content